### DISCUSSION TITLE
Possible to get gtag client_id and session_id for manual cross domain tracking?

### DISCUSSION THREAD

**Discussion by pixelpaulaus** [johannschopplich/nuxt-gtag#91] [General]
Hello,
With this module is it possible to get gtag client_id and session_id for manual cross domain tracking?

such as: https://support.google.com/analytics/answer/10071811?hl=en#zippy=%2Cmanual-setup

**Comment by johannschopplich** [comment]
Yes, absolutely. The recommended approach is to configure cross-domain measurement via the Analytics interface. However, there might be scenarios where you need to implement a custom solution via the measurement code. This is possible with this module. 

To retrieve a client ID on the source domain, you use the `useGtag` composable to retrieve the `gtag` instance and do whatever the Google docs offer:

```ts
const { gtag } = useGtag()

gtag('get', 'TAG_ID', 'client_id', (client_id) => {
  // Store the client ID in a variable.
});
gtag('get', 'TAG_ID', 'session_id', (session_id) => {
  // Store the session ID in a variable.
});
```

**Reply by pixelpaulaus** [reply]
i cant seem to get this to work. The client_id and session_id always seem to be empty/null.
But I do have the cookies in the browser and session. Is there anything else I need to add/set?
